internal/config.go

```diff
@@ -18,7 +19,8 @@ type IRCSettings struct {
 	Port                int      `env:"IRC_PORT" envDefault:"6667" validate:"min=0,max=65535"`
 	TLSAllowSelfSigned  bool     `env:"IRC_CERT_ALLOW_SELFSIGNED" envDefault:"true"`
 	TLSAllowCertExpired bool     `env:"IRC_CERT_ALLOW_EXPIRED" envDefault:"true"`
-	Channel             string   `env:"IRC_CHANNEL,required"`
+	Channel             string   `env:"IRC_CHANNEL,required" validate:"notempty"`
+	ChannelKey          string   `env:"IRC_CHANNEL_KEY" envDefault:""`
```

Comment:
This seems to be a part of another PR?

Reply:
Yes, it is. In this PR, I have not changed anything in this file.